tests/CMakeLists.txt

```diff
@@ -1,17 +1,19 @@
 cmake_minimum_required(VERSION 3.0.2)
```

Comment:
Not your code but this is inconsistent with the main CMakeLists.txt

cmake_minimum_required(VERSION 3.1.3...3.16)